File: CS/TemplateCardCaptionLocalization/MyLayoutViewDesignerHelper.cs
using DevExpress.XtraGrid.Views.Layout.Designer;
using DevExpress.XtraGrid.Views.Layout;
using TemplateCardCaptionLocalization.Properties;

namespace TemplateCardCaptionLocalization {
    public class MyLayoutViewDesignerHelper :LayoutViewDesignerHelper {
        public MyLayoutViewDesignerHelper(LayoutViewCustomizer customizer, LayoutView view)
            : base(customizer, view) { }

        public override void SynchronizeDesignerFromView(LayoutView view) {
            base.SynchronizeDesignerFromView(view);
            DesignerControl.Root.Text = Resources.LayoutViewDesignerHelper_DesigherControl_Root_Text;
        }
    }
}

File: VB/TemplateCardCaptionLocalization/MyLayoutViewDesignerHelper.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.XtraGrid.Views.Layout.Designer
Imports DevExpress.XtraGrid.Views.Layout

Namespace TemplateCardCaptionLocalization
	Public Class MyLayoutViewDesignerHelper
		Inherits LayoutViewDesignerHelper
		Public Sub New(ByVal customizer As LayoutViewCustomizer, ByVal view As LayoutView)
			MyBase.New(customizer, view)
		End Sub

		Public Overrides Sub SynchronizeDesignerFromView(ByVal view As LayoutView)
			MyBase.SynchronizeDesignerFromView(view)
            DesignerControl.Root.Text = Global.Resources.LayoutViewDesignerHelper_DesigherControl_Root_Text
		End Sub
	End Class
End Namespace